include/librealsense2/hpp/rs_device.hpp

```diff
@@ -879,7 +879,14 @@ namespace rs2
 
         void on_calibration_change( rs2_calibration_status status ) noexcept override
         {
-            _callback( status );
+            try
+            {
+                _callback( status );
+            }
+            catch( const std::exception & e )
+            {
+                LOG_DEBUG( "Error from callback: " << e.what() );
```

Comment:
This should be a warning (not debug level) so the user can see it.

Reply:
I removed LOG because it failed in GitHub actions constantly.